### ISSUE TITLE
Не добавляет фильтр в середине страницы

### ISSUE THREAD

**Issue by Shabanovv** [mindbox-moscow/ui-kit#577]
Если нахожусь не в самом верху страницы, а ниже - не дает добавить фильтр https://recordit.co/0KvavQZNZr

**Comment by Shabanovv** [comment]
При скролле вниз кнопки "добавить фильтр / или" должны скрываться

![image](https://user-images.githubusercontent.com/41429186/69425007-1254cb80-0d3b-11ea-9bb7-d7ba8c6c1185.png)

Вместо них дожна появиться кнопка "наверх" (по размеру как кнопка добавить фильтр) 

![image](https://user-images.githubusercontent.com/41429186/69425122-5f38a200-0d3b-11ea-8702-ece72fdbade5.png)

При нажатии - страница скроллится на самый верх

**Comment by timramone** [comment]
А когда у нас что-то набрано в фильтре, такое должно происходить?

**Comment by Shabanovv** [comment]
Да, если в фильтре есть условия, блок схлопываться (так ведь уже сейчас происходит)

**Comment by timramone** [comment]
Но кнопка вверх то не рисуется)

**Comment by Shabanovv** [comment]
А она может внезапно вжух* и появиться? 

*вжух = магия верстки, при которой одни кнопки внезапно исчезают, а другие - появляются
